docs/faq.md

```diff
@@ -344,6 +345,12 @@ See more at [Using Google Container Registry (GCR) with Minikube](https://ryanes
 
 Jib currently requires configuring your build tool to use the appropriate [Java networking properties](https://docs.oracle.com/javase/8/docs/api/java/net/doc-files/net-properties.html) (`https.proxyHost`, `https.proxyPort`, `https.proxyUser`, `https.proxyPassword`).
 
+### What should I do when the registry responds with Forbidden or DENIED?
+
+If the registry returns "403 Forbidden" or `"code":"DENIED"`, it often means Jib successfully authenticated using credentials you provided but the credentials do not have permissions to pull or push images. Make sure your account/role has the permissions to do the operation.
+
+However, depending on registry implementations, it is also possible that the registry actually meant you are not authenticated. Sometimes, it may be due to Jib picking up different credentials, credentials incorrectly configured, or typos in auth configuration. Or, it may be that your credentials work for pulling the base image but not for pushing. See [What should I do when the registry responds with UNAUTHORIZED?](#what-should-i-do-when-the-registry-responds-with-unauthorized) to ensure you have set up credentials correctly.
```

Comment:
```diff
- However, depending on ...
+ Depending on ...
```

```diff
- implementations, it is also possible ... 
+ implementations it is possible ...
```

I think you can remove the whole section about wrong credentials, and just directly link to the faq after `meant you are not authenticated.` There's not much value in repeating all the reasons for unauthorized again here.

Reply:
>```diff
>- implementations, it is also possible ... 
>+ implementations it is possible ...
>```

Actually I think this comma should be here.

Reply:
Retaining the comma.